tgui/packages/tgui/interfaces/EscapePodConsole.tsx

```diff
@@ -21,6 +22,9 @@ export const EscapePodConsole = (_props, context) => {
     case 4:
       statusMessage = 'NO EVACUATION';
       buttonColor = 'neutral';
+      if (data.launch_without_evac) {
+        operable = 1;
```

Comment:
not your fault but why are we using 0s and 1s as booleans as if this was DreamMaker